LocalStorage.js

```diff
@@ -0,0 +1,15 @@
+const formString = document.querySelector('.contact-me-form-button');
+if (formString) {
+  formString.addEventListener('click', () => {
+    const information = {
+      name: document.getElementById('name').value,
+      email: document.getElementById('email').value,
+      message: document.getElementById('message').value,
+    };
+    localStorage.setItem('formString', JSON.stringify(information));
+  });
```

Comment:
- [ ] Please ensure that the entered values are saved in the browser even if the form is not submitted. Therefore, I recommend saving the information after any changes are made to the input fields, rather than preserving it after the form submit button is clicked. :pray: You can read more about the input field change event [here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event). :blush: